modules/installation-azure-preparing-diskencryptionsets.adoc

```diff
@@ -0,0 +1,144 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/enabling-disk-encryption-sets-azure.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-disk-encryption-sets"]
+= Preparing an Azure Disk Encryption Set
+The {product-title} installer can use an existing Disk Encryption Set with a user-managed key. To enable this feature, you can create a Disk Encryption Set in Azure and provide the key to the installer.
+
+.Procedure
+
+. Set the following environment variables for the Azure resource group:
++
+[source,terminal]
+----
+$ export RESOURCEGROUP="<resource_group>"
+----
++
+[source,terminal]
+----
+$ export LOCATION="<location>"
+----
++
+where:
```

Comment:
I think this should be done in one of two ways. 
1. Use one command to create both env. variables, then use the numbered call outs I described earlier to apply these definitions, or 
2. Individual steps that explain the purpose behind each environment variable. 

There should only be one command per step.